modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectActionAPI.macro

```diff
@@ -0,0 +1,48 @@
+definition {
+
+	macro _curlObjectAction {
+		var portalURL = JSONCompany.getPortalURL();
+
+		if (isSet(objectDefinitionId)) {
+			var api = "object-definitions/${objectDefinitionId}/object-actions";
+		}
+
+		var curl = '''
+			${portalURL}/o/object-admin/v1.0/${api} \
+				-u test@liferay.com:test \
+				-H accept: application/json \
+				-H Content-Type: application/json \
+		''';
+
+		return ${curl};
+	}
+
+	macro createObjectAction {
+		Variables.assertDefined(parameterList = "${objectDefinitionId},${label},${name},${objectActionExecutorKey},${objectActionTriggerKey},${notificationTemplateId},${type},${notificationTemplateErc}");
+
+		var curl = ObjectActionAPI._curlObjectAction(objectDefinitionId = ${objectDefinitionId});
+		var body = '''
+            -d {
```

Comment:
Please check the sf. 
And also, if only one method, we can put body and curl in the upper _curlObjectAction(), but I'm ok if you don't want to. Just an suggestion to add api and body to the curl part.

Reply:
yeah I agree, we can put body and curl together here :)

Reply:
ok, I will merge them into one method.